pkg/controller/deployment/sync.go

```diff
@@ -24,7 +24,7 @@ import (
 	"strconv"
 
 	apps "k8s.io/api/apps/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Is it necessary?

Reply:
No, GitHub Codespaces kept on adding it but will remove it in a sec.

Sorry about that!